modules/openapi-generator/src/main/resources/Javascript/partial_model_generic.mustache

```diff
@@ -77,6 +77,13 @@
       {{/parentModel}}
       {{/useInheritance}}
       {{#vars}}
+      {{#requiredValidate}}
```

Comment:
I don't see any reference to `requiredValidate` in the code base. Where does this tag come from?

Reply:
additional-properties requiredValidate implementation addition